scripts/create_dev_meetings_report.py

```diff
@@ -33,8 +33,20 @@
 
 title = f'# Gammapy Developer Meeting \n'
 report_heading = f' * {today_long}, at 2 pm (CET) \n ' \
-                 '* Gammapy Developer Meeting on Zoom (direct link on Slack) \n' \
-                 '# Agenda\n'
+    '* Gammapy Developer Meeting on Zoom (direct link on Slack) \n' \
+    '# Agenda\n' \
+    '## General information\n\n' \
+    '## Open issues\n\n' \
+    '## Bugs\n\n' \
+    '## Documentation\n\n' \
+    '## DevOps\n\n' \
+    '## Validation & benchmark\n\n' \
+    '## Ongoing projects:\n\n' \
+    '### sensi\n\n' \
+    '### HLI\n\n' \
+    '### MWL\n\n' \
```

Comment:
You might remove this for now as this not clear what the projects will be for the next months